lib/util/common.js

```diff
@@ -41,10 +41,17 @@ function addHook (fastify) {
   }
 }
 
-function shouldRouteHide (schema, hiddenTag) {
+function shouldRouteHide (schema, hiddenTag, hideUntagged) {
   if (schema && schema.hide) {
     return true
   }
+
+  const tags = (schema && schema.tags) || []
+
+  if (tags.length === 0 && hideUntagged) {
+    return true
+  }
+
   if (schema && schema.tags && schema.tags.includes(hiddenTag)) {
```

Comment:
You have ensure the tags be an array in above of code. No need to check it like this in here. And `tags.includes(hiddenTag)` is duplicate work.
```suggestion
  if (tags.includes(hiddenTag)) {
    return true
```

Reply:
@climba03003 
how careless I am? I forgot. I will add this in PR after feedback on my question above.

Reply:
It is not your fault, it is the code in the pass. But with your changes, it be better to update together.